app/views/layouts/application.html.erb

```diff
@@ -48,9 +48,8 @@
 
     <% if join_path? || update_account_path? %>
       <!-- Google Maps API -->
-      <% google_maps_url = "https://maps.googleapis.com/maps/api/js?key=#{ENV['GOOGLE_API_KEY']}&libraries=places&callback=initAutocomplete" %>
-      <link rel="preconnect" href=<%= google_maps_url %>>
-      <%= javascript_include_tag google_maps_url, async: true %>
+      <% google_maps_url = "https://maps.googleapis.com/maps/api/js?key=#{ENV['GOOGLE_API_KEY']}&libraries=places" %>
```

Comment:
If it's preferred I can add this to the `InputLocation` component and lazy load it there, but I think the performance will be better with this load happening

Reply:
found a potentially better way of doing this by loading the google maps scripts via yarn, trying this out now

Reply:
If there is a better way to do this via yarn, we should also add this component to Storybook as well